res/database/Default/user/cujo.xml

```diff
@@ -1401,65 +1536,85 @@
+						<de>[0|Full] malt eine Windmühle vor einem Weizenfeld und erklärt den Zuschauern, wie einfach es ist die Getreideähren zu malen.</de>
+						<en>[0|Full] paints a windmill in front of a wheat field and explains to the audience how easy it is to paint the ears of grain.</en>
 					</description>
 					<ratings critics="29" speed="5" />
 				</programme>
-				<programme id="cujo-serien-spass-malen-04" product="2" licence_type="2" tmdb_id="0" imdb_id="" creator="8827" created_by="Cujo">
+				<programme id="cujo-serien-spass-malen-04" product="2" licence_type="2" tmdb_id="0" imdb_id="" fictional="1" creator="8827" created_by="Cujo">
 					<title>
 						<de>Dunkle Wolken am Himmel</de>
 						<en>Dark clouds in the sky</en>
 					</title>
+					<title_original>
+						<de>Dunkle Wolken am Himmel</de>
+						<en>Dark clouds in the sky</en>
+					</title_original>
 					<description>
-						<de>Blob Ross malt dunkle Wolken an einem Himmel und gibt den Zuschauern Tipps, wie sie ein Gewitter malen können.</de>
-						<en>Blob Ross paints dark clouds in a sky and gives viewers tips on how to paint a thunderstorm.</en>
+						<de>[0|Full] malt dunkle Wolken an einem Himmel und gibt den Zuschauern Tipps, wie sie ein Gewitter malen können.</de>
+						<en>[0|Full] paints dark clouds in a sky and gives viewers tips on how to paint a thunderstorm.</en>
 					</description>
 					<ratings critics="29" speed="5" />
 				</programme>
-				<programme id="cujo-serien-spass-malen-05" product="2" licence_type="2" tmdb_id="0" imdb_id="" creator="8827" created_by="Cujo">
+				<programme id="cujo-serien-spass-malen-05" product="2" licence_type="2" tmdb_id="0" imdb_id="" fictional="1" creator="8827" created_by="Cujo">
 					<title>
 						<de>Sonnenblumen am Wegesrand</de>
 						<en>Sunflowers by the wayside</en>
 					</title>
+					<title_original>
+						<de>Sonnenblumen am Wegesrand</de>
+						<en>Sunflowers by the wayside</en>
+					</title_original>
 					<description>
-						<de>Blob Ross malt drei Sonnenblumen am Wegesrand und zeigt das richtige Malen des Schattenwurfs.</de>
-						<en>Blob Ross paints three sunflowers by the wayside and demonstrates how to properly paint the shadow cast.</en>
+						<de>[0|Full] malt drei Sonnenblumen am Wegesrand und zeigt das richtige Malen des Schattenwurfs.</de>
+						<en>[0|Full] paints three sunflowers by the wayside and demonstrates how to properly paint the shadow cast.</en>
 					</description>
 					<ratings critics="29" speed="5" />
 				</programme>
 			</children>
 		</programme>
 		<!-- Doppelpass -->
-		<programme id="cujo-show-doppeltspass" product="3" licence_type="1" tmdb_id="0" imdb_id="tt0435952" creator="8827" created_by="Cujo">
+		<programme id="cujo-show-doppeltspass" product="3" licence_type="1" tmdb_id="0" imdb_id="tt0435952" creator="8827" created_by="Cujo" comment="TODO: It's a series. Create more episodes?">

```

Comment:
TODO rather "make it a script template"

Reply:
enhanced the comment.
it's a real non-fictional show though.

Reply:
...but I have an idea about an enhancement what to do about those, I'll post an issue / feature request.

Reply:
like this https://github.com/TVTower/TVTower/issues/957